common-libs/internal-api-clients/package-lock.json

```diff
@@ -38,6 +39,59 @@
         "typescript": "^4.3.2"
       }
     },
+    "node_modules/@affinidi/affinidi-did-auth-lib": {
+      "version": "1.0.8",
+      "resolved": "https://registry.npmjs.org/@affinidi/affinidi-did-auth-lib/-/affinidi-did-auth-lib-1.0.8.tgz",
+      "integrity": "sha512-v0RE14H+CLWZ4PwVNCX+xgF5Koqw14wdt8n1KoFQfiI440wduqZXMinDuwIopY40hZ0XxKTDaGBxvlTnntjJgQ==",
+      "dependencies": {
+        "@affinidi/common": "^1.3.1",
```

Comment:
We _really_ need to fix this someday...

(Probably by making `affinidi-did-auth-lib` a part of this monorepo)